features/gezag-minderjarige/een-ouder.feature

```diff
@@ -0,0 +1,76 @@
+#language: nl
+
+Functionaliteit: Gezag bepalen voor een minderjarige met één juridische ouder
+
+
+    Achtergrond:
+      Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Aart' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En de persoon 'Ariana' met burgerservicenummer '000000036'
+      * is meerderjarig
+      En de persoon 'Bert' met burgerservicenummer '000000048'
+      * is minderjarig
+      * heeft 'Gerda' als ouder
+
+
+  Regel: Als de minderjarige tijdens het huwelijk of partnerschap van de juridische ouder geboren is, dan hebben de ouder en diens (toenmalige) partner gezamenlijk gezag
+
+    Scenario: Minderjarige heeft ouder die gehuwd is met een meemoeder
+      Gegeven 'Gerda' en 'Ariana' zijn 7 jaar geleden gehuwd
+      En 'Bert' is 6 jaar geleden geboren
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
+      Dan is het antwoord 'GezamenlijkGezag'
+      * met ouder 'Gerda'
+      * met derde 'Ariana'
+
+    Scenario: Minderjarige heeft ouder die partnerschap heeft met een meemoeder
+      Gegeven 'Gerda' en 'Ariana' zijn 7 jaar geleden een geregistreerd partnerschap aangegaan
+      En 'Bert' is 6 jaar geleden geboren
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
+      Dan is het antwoord 'GezamenlijkGezag'
+      * met ouder 'Gerda'
+      * met derde 'Ariana'
+
+    Scenario: Minderjarige is geboren voor 1 april 2014 en moeder heeft geregistreerd partnerschap
```

Comment:
Willen we van deze scenario nog het variant vrouw-vrouw relatie?

> Minderjarige is geboren voor 1 april 2014 en moeder heeft geregistreerd partnerschap met een vrouw

Volgens de documentatie moet dit leiden tot eenhoofdig ouderlijk gezag, alhoewel het momenteel niet is geïmplementeerd.